index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TanmayiPortfolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <!--Outer div consists toolbar and headline-->
+    <div class="wrapper">
+        <!--section one toolbar-->
+        <header class="toolbar">
+          <nav class="logo-hamburger">
+            <a href='#' class="logo"><span>T</span></a>
+            <a href='#' class="hamburger"><img src="./media/hamburger.png" alt="menu"></a>
+          </nav>
+        </header>
+        <!--section two headline-->
+        <section class="headline">
+          <div class="primary text">
+            <h1>Hey there.
+                I'm Tanmayi.
+            </h1>
+            <h2 id="green">I'm a software developer</h2>
+          </div>
```

Comment:
Your design
![image](https://github.com/tan12082001/Project-portfolio/assets/78251927/33f0a3fc-c246-4b17-bb3f-313808266abf)

Figma
![image](https://github.com/tan12082001/Project-portfolio/assets/78251927/5f800416-da40-46f0-ac4c-c8efe4e24d8c)

According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), to ensure that our website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). kindly adjust the layout of this section. 💪🛠